pkg/vpn/connect.go

```diff
@@ -15,6 +16,16 @@ import (
 	"tailscale.com/tsnet"
 )
 
+type ConnectOpt interface{}
+
+type connectOptOutputWriter struct {
+	out io.Writer
+}
+
+func ConnectionOptOutputWriter(out io.Writer) ConnectOpt {
```

Comment:
Maybe 

```suggestion
func WithOutputWriter(out io.Writer) ConnectOpt {
```

Reply:
The `With...` functions I typically use for chaining constructors, but this is a bit different as options are not meant to be chained, so I find it a bit misleading.